cardano_node_tests/tests/test_plutus_spend_build.py

```diff
@@ -1562,6 +1562,133 @@ def test_guessing_game(
 
         assert "The Plutus script evaluation failed" in str(excinfo.value)
 
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    def test_two_scripts_spending_one_fail(
+        self,
+        cluster: clusterlib.ClusterLib,
+        payment_addrs: List[clusterlib.AddressRecord],
+    ):
+        """Test locking two Tx outputs with two different Plutus scripts in single Tx, one fails.
+
+        Uses `cardano-cli transaction build` command for building the transactions.
+
+        * create a Tx output with a datum hash at the script addresses
+        * try to spend the locked UTxOs
+        * check that the expected error was raised
+        """
+        temp_template = common.get_test_id(cluster)
+        amount = 50_000_000
+
+        script_fund = 500_000_000
+        collateral_fund = 750_000_000
```

Comment:
You can compute minimum necessary collateral amount the same way that is now used in the rest of the tests, e.g. https://github.com/input-output-hk/cardano-node-tests/blob/master/cardano_node_tests/tests/test_plutus_spend_build.py#L713-L715